release-notes/README.md

```diff
@@ -4,7 +4,23 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
-## March 3, 2021
+## March 4, 2022
+
+### ACP Xamarin frameworks
+
+Updated the Android and iOS native libraries for the [ACPCore](https://github.com/adobe/xamarin-acpcore) and [ACPAnalytics](https://github.com/adobe/xamarin-acpanalytics) Xamarin frameworks:
+
+* [Xamarin Core Android 1.1.0](https://www.nuget.org/packages/Adobe.ACPCore.Android/1.1.0) : Android Core updated to 1.10.0
```

Comment:
nit: can you remove the extra space before column for all occurrences below, e.g:
[Xamarin Core Android 1.1.0](https://www.nuget.org/packages/Adobe.ACPCore.Android/1.1.0): Android Core updated to 1.10.0

Reply:
Removing the space was messing up the link when I tried it in Xcode editor, let me give it a try